packages/odyssey-react-mui/src/labs/SideNav/SideNavItemContent.tsx

```diff
@@ -167,6 +100,67 @@ const SideNavItemContent = ({
 
   const odysseyDesignTokens = useOdysseyDesignTokens();
 
+  const navItemContentStyles = useMemo(
+    () => ({
+      display: "flex",
+      alignItems: "center",
+      width: "100%",
+      textDecoration: "none",
+      color: `${isDisabled ? odysseyDesignTokens.TypographyColorDisabled : odysseyDesignTokens.TypographyColorHeading} !important`,
+      minHeight: isCompact
+        ? odysseyDesignTokens.Spacing6
+        : odysseyDesignTokens.Spacing7,
+      padding: isCompact
+        ? `${odysseyDesignTokens.Spacing0} ${odysseyDesignTokens.Spacing4}`
+        : `${odysseyDesignTokens.Spacing2} ${odysseyDesignTokens.Spacing4}`,
+      "&:focus-visible": {
+        borderRadius: 0,
+        outlineColor: odysseyDesignTokens.FocusOutlineColorPrimary,
+        outlineStyle: odysseyDesignTokens.FocusOutlineStyle,
+        outlineWidth: odysseyDesignTokens.FocusOutlineWidthMain,
+        textDecoration: "none",
+        outlineOffset: 0,
+        backgroundColor: !isDisabled
+          ? odysseyDesignTokens.HueNeutral50
+          : "inherit",
+        color: `${odysseyDesignTokens.TypographyColorAction} !important`,
+      },
+      "&:hover": {
+        textDecoration: "none",
+        cursor: isDisabled ? "default" : "pointer",
+        color: isDisabled
+          ? "default"
+          : `${odysseyDesignTokens.TypographyColorAction} !important`,
+        backgroundColor: !isDisabled
+          ? odysseyDesignTokens.HueNeutral50
+          : "inherit",
+      },
+    }),
+    [odysseyDesignTokens, isCompact, isDisabled],
+  );
+
+  const NavItemContentContainer = styled("div", {
+    shouldForwardProp: (prop) =>
+      prop !== "odysseyDesignTokens" &&
+      prop !== "isCompact" &&
+      prop !== "isDisabled",
+  })<{
+    disabled?: boolean;
+  }>(({}) => ({
+    ...navItemContentStyles,
+  }));
```

Comment:
I hope I didn't mark this wrong in an earlier comment. If so, I apologize. These styles and the creation of `NavItemContentContainer` can be statically assigned above the component, so you won't need `useMemo`.